src/db/file/pushes.ts

```diff
@@ -51,6 +51,18 @@ export const getPush = async (id: string) => {
   });
 };
 
+export const deletePush = async (id: string) => {
+  return new Promise<void>((resolve, reject) => {
+    db.remove({ id }, (err) => {
+      if (err) {
+        reject(err);
```

Comment:
I ignore lines like this in #979 for coverage as its hard to get neDB to actually throw an error without corrupting the database file.